### DISCUSSION TITLE
Table Title has weird overlay effect

### DISCUSSION THREAD

**Discussion by krubenok** [community/community#13216] [Other Feature Feedback, Questions, & Ideas]
<img width="173" alt="image" src="https://user-images.githubusercontent.com/2529120/159415781-0099bcf1-b982-493f-8a86-ef061c5b1419.png">
 see screenshot, when active there's a weird hall of mirrors looking effect here...

**Comment by jfullstackdev** [comment]
is this fixed by now ? never had that on my end
